apps/ideas/views.py

```diff
@@ -109,6 +109,11 @@ class IdeaCreateView(AbstractIdeaCreateView):
     permission_required = 'a4_candy_ideas.add_idea'
     template_name = 'a4_candy_ideas/idea_create_form.html'
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs['user'] = self.request.user
+        return kwargs
+
```

Comment:
Yes, it was there first, but then all the tests are failing because the mixin is not yet there. We can either add it to all forms right away or keep it like that, test in idea on dev and then move it back when we add it to all the other modules.

Reply:
Sure, we can also move it there later, didn't think about failing tests!